runhttpd.sh

```diff
@@ -1,11 +1,19 @@
 #!/usr/bin/bash
 
+IP=${IP:-"172.22.0.1"}
 HTTP_PORT=${HTTP_PORT:-"80"}
 
-rmdir /var/www/html
-ln -s /shared/html/ /var/www/html
+mkdir -p /shared/html
+
+# Copy files to shared mount
+cp /tmp/inspector.ipxe /shared/html/inspector.ipxe
+cp /tmp/dualboot.ipxe /shared/html/dualboot.ipxe
+
+# Use configured values
+sed -i -e s/IRONIC_IP/$IP/g -e s/HTTP_PORT/$HTTP_PORT/g /shared/html/inspector.ipxe
 
 sed -i 's/^Listen .*$/Listen '"$HTTP_PORT"'/' /etc/httpd/conf/httpd.conf
+sed -i 's|\(^[[:space:]]*\)\(DocumentRoot\)\(.*\)|\1\2 /shared/html/|' /etc/httpd/conf/httpd.conf
```

Comment:
In addition to changing DocumentRoot, also need to change access permissions for that dir, e.g. this config in http.conf:
# Further relax access to the default document root:
<Directory "/var/www/html">

which is why the global replace of /var/www/html works.

Otherwise get:
HTTP request sent, awaiting response... 403 Forbidden
2019-04-01 10:50:17 ERROR 403: Forbidden.